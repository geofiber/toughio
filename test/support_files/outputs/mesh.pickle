version https://git-lfs.github.com/spec/v1
oid sha256:4a8f4e60e01fe9ac411f2a170e69231a8b70c6024de83fe29b9f6f63c17fa14c
size 145686
