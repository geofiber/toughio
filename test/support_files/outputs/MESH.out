version https://git-lfs.github.com/spec/v1
oid sha256:10ae6888cde890d036d51fbf52bcd7fecca6fb8f2c3b542b6ba46d12ec85dcd2
size 216501
